pkgs/development/python-modules/buildbot/pkg.nix

```diff
@@ -2,11 +2,11 @@
 
 buildPythonPackage rec {
   pname = "buildbot-pkg";
-  version = "2.8.0";
+  version = "2.8.1";
```

Comment:
Wouldn't it make sense to pin this to `buildbot.version`?

Reply:
Yeah, I think that's a good idea. It'll prevent buildbot from being updated without updating the plugins (because the plugins inherit their version from `buildbot-pkg`)